src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
make colorSuplier a class level field, it is not okay to create a new object per method call